config/openshift/kustomization.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../default
+- privileged-role-binding.yaml
+patches:
```

Comment:
Nit: can we move these patches into separate files as similar to [bgpTypeFRR.yaml](https://github.com/metallb/metallb-operator/blob/main/config/frr-on-ci/bgpTypeFRR.yaml) ?

Reply:
done